src/sentry/static/sentry/app/views/releases/detail/overview/chart/releaseChartControls.tsx

```diff
@@ -186,6 +183,61 @@ const ReleaseChartControls = ({
   );
 };
 
+const eventTypeOptions: SelectValue<EventType>[] = [
+  {value: EventType.ALL, label: t('All')},
+  {value: EventType.CSP, label: t('CSP')},
+  {value: EventType.DEFAULT, label: t('Default')},
+  {value: EventType.ERROR, label: 'Error'},
+  {value: EventType.TRANSACTION, label: t('Transaction')},
+];
+
+const vitalTypeOptions: SelectValue<WebVital>[] = [
+  WebVital.FP,
+  WebVital.FCP,
+  WebVital.LCP,
+  WebVital.FID,
+  WebVital.CLS,
+].map(vital => ({value: vital, label: WEB_VITAL_DETAILS[vital].name}));
+
+type SecondarySelectorProps = {
+  yAxis: YAxis;
+  eventType: EventType;
+  onEventTypeChange: (v: EventType) => void;
+  vitalType: WebVital;
+  onVitalTypeChange: (v: WebVital) => void;
+};
+
+function SecondarySelector({
+  yAxis,
+  eventType,
+  onEventTypeChange,
+  vitalType,
+  onVitalTypeChange,
+}: SecondarySelectorProps) {
+  switch (yAxis) {
+    case YAxis.EVENTS:
+      return (
+        <OptionSelector
+          title={t('Event Type')}
+          selected={eventType}
+          options={eventTypeOptions}
+          onChange={onEventTypeChange as (value: string) => void}
+        />
+      );
+    case YAxis.COUNT_VITAL:
+      return (
+        <OptionSelector
+          title={t('Vital Type')}
```

Comment:
```suggestion
          title={t('Vital')}
```

Do we refer to vitals as 'types' elsewhere?